plugins/tiddlywiki/multiwikiserver/modules/startup.js

```diff
@@ -44,7 +42,7 @@ function setupStore() {
 	return store;
 }
 
-function ServerManager(store) {
+function ServerManager() {
```

Comment:
I think removing it, is not right. But @Jermolene there is an inconsistency. Here it says: `ServerManager(store)` but in line 25 it says: 

```
	serverManager: new ServerManager({
			store: store
		})
```
Which seems to be an options object.

Reply:
The option is not used anywhere. Store is passed in somewhere else, not here.